main.html

```diff
@@ -1,77 +1,255 @@
+  This methodology provides a low-friction process for ensuring that the features and functionality of shared projects are responsive and nimble to the changing needs of their many internal consumers.
+</p>
 
-import seaborn as sns
-sns.set()
-plt.plot(x, np.sin(x), x, np.cos(x));
-</pre>
-<h2>Interactive widgets</h2>
-<pre data-code-language="python" data-executable="true" data-type="programlisting">%matplotlib inline
-from IPython.display import Image
-from IPython.html.widgets import interact
-from numpy import pi, cos, sin
-import numpy as np
-import pylab as plt
-import seaborn as sns
-sns.set()
-</pre>
-<pre data-code-language="pyt" data-executable="true" data-type="programlisting">def plot_fringe(bl_length, wavelength):
-    &quot;&quot;&quot; Plot the fringe function for a baseline (see Fig 1)
-
-    bl_length:      distance between antennas, in m
-    wavelength:     wavelength, in m
-    &quot;&quot;&quot;
-    theta = np.linspace(-np.pi, np.pi, 401)
-    l = sin(theta)
-    F = cos(2 * pi * bl_length * l / wavelength)
-    F2 = cos(2 * pi * 2 * bl_length * l / wavelength)
-
-    plt.plot(l, F, c='#cc0000', label=&quot;Baseline 1-2&quot;)
-    plt.plot(l, F2, c='#0000cc', label=&quot;Baseline 1-3&quot;)
-    plt.xlabel(&quot;$sin(\\theta)$&quot;)
-    plt.ylabel(&quot;Fringe amplitude&quot;)
-    plt.ylim(-2, 2)
-    plt.legend()
-
-f = interact(plot_fringe, bl_length=(1, 100), wavelength=(1, 100))
-&lt;/pre&gt;
+<p>
+  In this Oriole, we'll define some key terms used when discussing InnerSource.
+  We'll walk through the high-level mechanics of how InnerSourcing can work and enumerate the key principles upon-which these mechanics are based.
+  Finally, we'll share other references that can help you deepen your understanding and involvement in the craft of InnerSource.
+</p>
+
+<h2>Context</h2>
+
+<p>
+  Imagine two teams delivering separate pieces of software with one team's software depending on that of the other.
+  This situation is common in a large enterprise where a single team producing software may have dozens or hundreds of consumers.
+  When consuming teams need many features, producing teams normally have some sort of requirements and prioritization process to decide which features they will work on.
+  For feature requests that are not prioritized, the consuming team is faced with 3 choices, each of which come with their own drawbacks.
+</p>
+
+<ol>
+  <li>
+    <p>
+      <strong>Wait it out</strong>.
+      In this option, the consuming team does nothing and limps along without the requested functionality.
+      Depending on the benefit of the feature request, <em>Wait it out</em> might be just fine or it may come with real amounts of pain for the consuming team. 
+    </p>
+  </li>
+  <li>
+    <p>
+      <strong>Workaround</strong>.
+      A consuming team that doesn't want to wait may do extra work somewhere else to compensate for the absence of their requested feature.
```

Comment:
In looking at the html it looks fine without this change.  I would agree since you didn't come out and define it, it doesn't need to be highlighted in this way.